charms/kserve-controller/src/templates/configmap_manifests.yaml.j2

```diff
@@ -47,159 +54,46 @@ data:
     }
   ingress: |-
     {
-        "ingressGateway" : "kubeflow/kubeflow-gateway",
-        "ingressService" : "istio-ingressgateway-workload.kubeflow.svc.cluster.local",
-        "localGateway" : "kubeflow/kubeflow-gateway",
-        "localGatewayService" : "istio-ingressgateway-workload.kubeflow.svc.cluster.local",
-        "ingressDomain"  : "example.com"
+      "ingressGateway" : "kubeflow/kubeflow-gateway",
+      "ingressService" : "istio-ingressgateway-workload.kubeflow.svc.cluster.local",
+      "localGateway" : "kubeflow/kubeflow-gateway",
+      "localGatewayService" : "istio-ingressgateway-workload.kubeflow.svc.cluster.local",
```

Comment:
Should these be configured via templating?  They look like things that depend on how the user deployed.

...I realise this section is changed here only because of a formatting change, not that the settings are being changed.  To resolve this conversation for the present PR, opening and linking an issue works

Reply:
Yeah, this was not part of the first design of kserve, but we should take care of it. #82 tracks this.